opm/simulators/wells/BlackoilWellModelGeneric.cpp

```diff
@@ -1124,6 +1123,9 @@ updateNetworkPressures(const int reportStepIdx)
                 if (thp_is_limit) {
                     ws.thp = well->getTHPConstraint(summaryState_);
                 }
+                //PJPE: Set thp of wells belonging to a subsea manifold equal to the node_pressure
```

Comment:
Here, I believe the `dynamic_thp_limit_` is already updated with `well->setDynamicThpLimit(new_limit);` , so this code should not be necessary.  But  as communicated in the private email,  because of choke pressure, the 
`dynamic_thp_limit_` might not be the same with the nodal pressure of the node anymore.   It might be the nodal_pressure + choke_pressure.

Reply:
> so this code should not be necessary

Is removed

> `dynamic_thp_limit_` might not be the same with the nodal pressure of the node anymore

True, see my comment below.